fblldbbase.py

```diff
@@ -63,7 +63,7 @@ def evaluateIntegerExpression(expression, printErrors=True):
     output = output[2:]
   elif output.startswith('\\'): # Or as \0 (Dec)
     output = output[1:]
-  return int(output, 16)
+  return int(output, 10)
```

Comment:
Good point. I agree it would be better to make this `int(output, 0)` to handle any integer literal.

```
Python 3.4.3 (default, Jul 13 2015, 12:18:23)
[GCC 4.2.1 Compatible Apple LLVM 6.1.0 (clang-602.0.53)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> int('0o13217', 0)
5775
>>> int('0xf3217', 0)
995863
>>> int('0b1010110', 0)
86
>>> int('123', 0)
123
>>>
```